windows/RNFS/RNFS.vcxproj

```diff
@@ -60,6 +60,7 @@
     <PlatformToolset>v140</PlatformToolset>
     <PlatformToolset Condition="'$(VisualStudioVersion)' == '15.0'">v141</PlatformToolset>
     <PlatformToolset Condition="'$(VisualStudioVersion)' == '16.0'">v142</PlatformToolset>
+    <PlatformToolset Condition="'$(VisualStudioVersion)' == '17.0'">v143</PlatformToolset>
```

Comment:
I have seen some instances just fully omit all <PlatformToolset> instead of keep adding here. Doesn't it fallback on whatever version opened visual studio instead of having to keep specifying this?